.travis/Dockerfile

```diff
@@ -26,10 +26,10 @@ RUN cd /build && \
 	git clone -b $(curl -s -L http://grpc.io/release) https://github.com/grpc/grpc && \
 	cd /build/grpc && \
 	git submodule update --init && \
-	make -j && \
+	make -j4 && \
```

Comment:
Quick question on this -- have you run into VM hanging issues with higher levels of parallelism in builds?  I seem to get hangs when going above j2 (it's nondeterministic).